plop-templates/basic-component/docs/ComponentName.stories.tsx.hbs

```diff
@@ -0,0 +1,59 @@
+import React from "react"
+import { ComponentMeta, Story } from "@storybook/react"
+import { withDesign } from "storybook-addon-designs"
+import { CATEGORIES } from "../../../storybook/constants"
+import { figmaEmbed } from "../../../storybook/helpers"
+import { StoryWrapper } from "../../../storybook/components/StoryWrapper"
+import { {{pascalCase componentName}} } from "../index"
+
+export default {
+  title: `${CATEGORIES.components}/{{pascalCase componentName}}`,
+  component: {{pascalCase componentName}},
+  parameters: {
+    docs: {
+      description: {
+        component:
+          'import { {{pascalCase componentName}} } from "@kaizen/{{kebabCase componentName}}".',
+      },
+    },
+    ...figmaEmbed("REPLACE_THIS_WITH_FIGMA_URL"), /** @todo: Replace with Figma frame url */
+  },
+  decorators: [withDesign],
+} as ComponentMeta<typeof {{pascalCase componentName}}>
+
+export const DefaultKaizenSiteDemo = args => <{{pascalCase componentName}} {...args} />
+DefaultKaizenSiteDemo.storyName = "Default (Kaizen Site Demo)"
```

Comment:
I think we have decided that the naming of the demo story should just be the component name rather than Default (Kaizen Site Demo).

Reply:
Would there be away to create the .mdx file for the component within the /site/components folder? and from there that demo story url/id would link up to the `demoStoryId` parameter - I believe the previous generator created this but also this was when the .mdx was within the component folder...What are your thoughts?

Reply:
>I think we have decided that the naming of the demo story should just be the component name rather than Default (Kaizen Site Demo).

So would it look something like this?

```suggestion
DefaultKaizenSiteDemo.storyName = "Component Name" // title case
```

Or rename the story itself from `DefaultKaizenSiteDemo` to `ComponentName`?

Although I wonder if that would look a bit redundant in the tree since the directory itself is the component name 🤔 

>Would there be away to create the .mdx file for the component within the /site/components folder? and from there that demo story url/id would link up to the demoStoryId parameter - I believe the previous generator created this but also this was when the .mdx was within the component folder...What are your thoughts?

We absolutely can! :) Good suggestion. I had completely forgot about it. Ideally we'd also have a proper README once we remove the magic that is currently turning them into pages.

Reply:
> I think we have decided that the naming of the demo story should just be the component name rather than Default (Kaizen Site Demo).

But would that be on the story with the args exposed or just a base example that we would reference in the mdx docs? Just wondering if we need some kind of notation to describe that it is a demo in storybook